ansible/library/conn_graph_facts.py

```diff
@@ -235,14 +235,19 @@ def parse_graph(self):
                     for consolelink in allconsolelinks:
                         start_dev = consolelink.attrib['StartDevice']
                         end_dev = consolelink.attrib['EndDevice']
+                        try:
+                            console_proxy = consolelink.attrib['Proxy']
+                            console_type = consolelink.attrib['Console_type']
```

Comment:
Please also update `sonic_lab_console_links.csv` if the `Proxy` and `Console_type` are new fields to be added.

Reply:
done